lms/static/scripts/frontend_apps/components/LMSGrader.js

```diff
@@ -103,29 +88,45 @@ export default function LMSGrader({
   };
 
   return (
-    <div className="LMSGrader">
-      <header>
-        <ul className="LMSGrader__grading-components">
-          <li className="LMSGrader__title">
-            <h1 className="LMSGrader__assignment">{assignmentName}</h1>
-            <h2 className="LMSGrader__name">{courseName}</h2>
-          </li>
-          <li className="LMSGrader__student-picker">
-            <div className="LMSGrader__student-count">
-              {renderStudentCount()}
-            </div>
+    <>
+      <header
+        className={classNames(
+          'grid grid-cols-1 items-center gap-y-2 p-2',
+          'lg:grid-cols-3 lg:gap-x-4 lg:px-3'
+        )}
+      >
+        <div>
+          <h1 className="text-lg p-0 m-0" data-testid="assignment-name">
```

Comment:
The `p-0` and `m-0` here and in the following `h2` are side effects of LMS not having any reset styles.